default.xml

```diff
@@ -8,20 +8,20 @@
   <remote fetch="http://git.yoctoproject.org" name="yocto"/>
   <remote fetch="https://source.codeaurora.org/external" name="CAF"/>
 
-  <default revision="master" sync-j="4"/>
+  <default revision="zeus" sync-j="4"/>
```

Comment:
This doesn't look right to me.

Reply:
Good catch.
I accidentally added the d168e60 commit. Just removed and force-pushed.